package.json

```diff
@@ -103,10 +103,11 @@
     "precommit": "lint-staged",
     "semantic-release": "semantic-release pre && npm publish && semantic-release post",
     "start:storybook": "start-storybook -p 9001 -s ./storybook/static -c storybook/config",
-    "start:webpack": "yarn build -- --watch",
+    "start:webpack": "yarn webpack -- --watch",
```

Comment:
Oooh but the whole point is that we are testing dev mode now lol, hmmm maybe we can set the babel aliases for dev and prod to match, then 👍